requirements.txt

```diff
@@ -55,7 +55,7 @@ rsa==3.3
 s3transfer==0.0.1
 six==1.15.0
 termcolor==1.1.0
-troposphere==1.5.0
+troposphere==4.0.2
```

Comment:
This dependency update should already be in master. You could try to fetch and merge